### PR TITLE
Add the user attributes section and tenant domain infor section  to the try it out section in org login

### PR DESCRIPTION
**Pull request by Achintha444**

### CODE REVIEW

File: en/docs/guides/organization-login/try-organization-login.md [hunk 1]
```diff
@@ -67,6 +67,14 @@ To configure the business applications:
 
 7. Click **Update** to save the configurations.
 
+8.  Go to the User Attributes tab, and click on + Add User Attributes.
```
Comment (Sachin-Mamoru) [inline en/docs/guides/organization-login/try-organization-login.md:70]:
```suggestion
8.  Go to the **User Attributes** tab, and click on + Add User Attributes.
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 2]
```diff
@@ -67,6 +67,14 @@ To configure the business applications:
...
+    - Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list.
+
+    <img src="../../../assets/img/guides/organization-login/try-it-out/app_add_user_attributes.png" alt="App User Attributes Configurations" width="700" style="border:1px solid grey">
```
Comment (Sachin-Mamoru) [inline en/docs/guides/organization-login/try-organization-login.md:73]: Shall we remove the black background cover and show only the select user attributes part?

Comment (Achintha444) [reply en/docs/guides/organization-login/try-organization-login.md:73]: Sure, I will change it.

**Review by Sachin-Mamoru — APPROVED** [review]

File: en/docs/guides/organization-login/try-organization-login.md [hunk 3]
```diff
@@ -67,6 +67,14 @@ To configure the business applications:
 
 7. Click **Update** to save the configurations.
 
+8.  Go to the **User Attributes** tab, and click on + Add User Attributes.
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:70]:
```suggestion
8.  Go to **User Attributes** and click on **+ Add User Attributes**.
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 4]
```diff
@@ -67,6 +67,14 @@ To configure the business applications:
...
+    - Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list.
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:71]: Shall we add this as a separate step? step 9?

File: en/docs/guides/organization-login/try-organization-login.md [hunk 5]
```diff
@@ -67,6 +67,14 @@ To configure the business applications:
...
+
+    <img src="../../../assets/img/guides/organization-login/try-it-out/app_add_user_attributes.png" alt="App User Attributes Configurations" width="700" style="border:1px solid grey">
+
+9.  Click Save to update the configuration.
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:75]:
```suggestion
9.  Click **Save** to add the user attributes and click **Update** to save all the configurations.
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 6]
```diff
@@ -182,6 +190,9 @@ To set up the sample application:
     ]
     ```
 
+    !!! info
+        If the created application resides in a different organization, change the value in the ```WSO2IS_TENANT_NAME``` to that organization id.
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:194]: If the users are just reading through the doc before testing this out, they might wonder where `WSO2IS_TENANT_NAME` is. 

We better add it to the sample.

Comment (Achintha444) [reply en/docs/guides/organization-login/try-organization-login.md:194]: Sure, I will update the document

File: en/docs/guides/organization-login/try-organization-login.md (hunk 4 above)
Comment (Achintha444) [reply en/docs/guides/organization-login/try-organization-login.md:71]: Sure, I will update the document.

Comment (Achintha444) [reply en/docs/guides/organization-login/try-organization-login.md]: Updated the PR.

File: en/docs/guides/organization-login/try-organization-login.md (hunk 6 above)
Comment (Achintha444) [reply en/docs/guides/organization-login/try-organization-login.md:197]: Updated the PR.

File: en/docs/guides/organization-login/try-organization-login.md [hunk 7]
```diff
@@ -67,6 +67,15 @@ To configure the business applications:
 
 7. Click **Update** to save the configurations.
 
+8.  Go to **User Attributes** and click on **+ Add User Attributes**.
+
+9.  Select <i>Email, First Name, Last Name</i> and <i>Username</i> from the list..
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:72]:
```suggestion
9.  Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attributes.
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 8]
```diff
@@ -162,12 +171,14 @@ To set up the sample application:
     | Property  | Description   |
     |-----------|---------------|
     | WSO2IS_HOST   | The URL of the Identity Server.    |
+    | WSO2IS_TENANT_NAME   | Id of the Organization, that the created application resides in. <br> Default value for this proprty is ```carbon.super```   |
     | WSO2IS_CLIENT_ID  | The client ID obtained when creating an application on the Identity Server Console.    |
     | WSO2IS_CLIENT_SECRET  | The client secret obtained when creating an application on the Identity Server Console.   |
     | SAMPLE_ORGS   | The details of the organization. <br> `id`: ID of the sub-organizations using this shared application. |
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:177]:
```suggestion
    | WSO2IS_TENANT_NAME   | ID of the organization where the created application resides. <br> Default value for this property is `carbon.super`.   |
    | WSO2IS_CLIENT_ID  | The client ID obtained when creating an application on the Identity Server Console.    |
    | WSO2IS_CLIENT_SECRET  | The client secret obtained when creating an application on the Identity Server Console.   |
    | SAMPLE_ORGS   | The details of the organization. <br> `id`: ID of the sub-organizations using this shared application. |
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 9]
```diff
@@ -162,12 +171,14 @@ To set up the sample application:
     | Property  | Description   |
     |-----------|---------------|
     | WSO2IS_HOST   | The URL of the Identity Server.    |
+    | WSO2IS_TENANT_NAME   | Id of the Organization, that the created application resides in. <br> Default value for this proprty is ```carbon.super```   |
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:174]:
```suggestion
    | WSO2IS_TENANT_NAME   | ID of the organization where the created application resides. <br> Default value for this property is `carbon.super`.   |
```

File: en/docs/guides/organization-login/try-organization-login.md [hunk 10]
```diff
@@ -182,6 +193,9 @@ To set up the sample application:
     ]
     ```
 
+    !!! info
+        If the created application resides in a different organization, change the value in the ```WSO2IS_TENANT_NAME``` to that organization id.
```
Comment (divyaamunugama) [inline en/docs/guides/organization-login/try-organization-login.md:197]:
```suggestion
        If the created application resides in a different organization, replace the `WSO2IS_TENANT_NAME` value with the respective organization ID.
```

**Review by divyaamunugama — APPROVED** [review]

**Pull request merged**